surefire-its/src/test/java/org/apache/maven/surefire/its/UnicodeTestNamesIT.java

```diff
@@ -62,14 +65,13 @@ public void checkFileNamesWithUnicode()
                 unpacked.executeTest()
                         .assertTestSuiteResults( 2, 0, 0, 0 );
 
-        TestFile surefireReportFile = outputValidator.getSurefireReportsFile( TXT_REPORT );
+        TestFile surefireReportFile = outputValidator.getSurefireReportsFile( TXT_REPORT, UTF_8 );
         assertTrue( surefireReportFile.exists() );
-        surefireReportFile.assertContainsText( "junit.twoTestCases.????Test" );
+        surefireReportFile.assertContainsText( toUTF8( REPORT_FILE_CONTENT ) );
```

Comment:
@Tibor17 isnt's obvious that
toUTF8( REPORT_FILE_CONTENT ) not equals(REPORT_FILE_CONTENT)

IMHO we could write a comment to explain why this toUTF8 is needed, or add a
assertNotEquals(REPORT_FILE_CONTENT, toUTF8( REPORT_FILE_CONTENT ))